Library/Homebrew/cask/lib/hbc/cli/style.rb

```diff
@@ -11,7 +11,7 @@ def self.help
 
       def run
         install_rubocop
-        system "rubocop", *rubocop_args, "--", *cask_paths
+        system({ "XDG_CACHE_HOME" => HOMEBREW_CACHE }, "rubocop", *rubocop_args, "--", *cask_paths)
```

Comment:
But it still doesn't make a difference when we override it with `HOMEBREW_CACHE` anyway for RuboCop.